content/docs/other-guides/multi-user-overview.md

```diff
@@ -6,9 +6,9 @@ weight = 50
 
 {{% stable-status %}}
 
-As of v0.6, Kubeflow supports for multi-user isolation of user-created
+As of v0.6, Kubeflow added multi-user support which apply access control over namespaces / user-created
```

Comment:
Is "namespaces / user-created " a typo? I don't really understand what the "/" means